zkevm-circuits/src/table.rs

```diff
@@ -809,6 +836,10 @@ impl MptTable {
         offset: usize,
         row: &MptUpdateRow<Value<F>>,
     ) -> Result<(), Error> {
+        if offset == 4 || offset == 5 || offset == 7 || offset == 8 {
```

Comment:
why are these offsets special ?

Reply:
they were there for debugging. they're the mpt updates where the value is non-zero.